jerry-ext/CMakeLists.txt

```diff
@@ -47,5 +47,9 @@ target_include_directories(${JERRY_EXT_NAME} PRIVATE ${INCLUDE_EXT_PRIVATE})
 target_compile_definitions(${JERRY_EXT_NAME} PUBLIC ${DEFINES_EXT})
 target_link_libraries(${JERRY_EXT_NAME} jerry-core)
 
+if(USING_MSVC AND FEATURE_DEBUGGER)
+  target_link_libraries(${JERRY_EXT_NAME} ws2_32.lib)
```

Comment:
Let' s state it upfront, I'm not completely familiar with cmake on windows (although I had to play with it a bit): but is the `.lib` suffix/extension needed for `target_link_libraries`? Isn't that function working with "library names" and appends/prepends whatever is mandated by the platform/os? (E.g., name "xxx" gets converted to `libxxx.a`, `libxxx.so`, `xxx.lib`, `xxx.dll`, etc. as needed?)

Reply:
True, there is no need for the `.lib` extension.